src/components/ResourceBrowser/Utils.tsx

```diff
@@ -350,17 +350,15 @@ export const getRenderNodeButton =
         handleNodeClick: (e: React.MouseEvent<HTMLButtonElement>) => void,
     ) =>
     (children: React.ReactNode) => (
-        <Tippy className="default-tt" arrow={false} placement="top" content={resourceData[columnName]}>
-            <button
-                type="button"
-                className="dc__unset-button-styles dc__ellipsis-right dc__block"
-                data-name={resourceData[columnName]}
-                onClick={handleNodeClick}
-                aria-label={`Select ${resourceData[columnName]}`}
-            >
-                <span className="dc__link">{children}</span>
-            </button>
-        </Tippy>
+        <button
+            type="button"
+            className="dc__unset-button-styles dc__no-decor flex"
```

Comment:
Check if we need dc__truncate?

Reply:
the child being passed in here is a span with dc__truncate applied on it